### PR TITLE
Add type definitions for mokapi package

### PR DESCRIPTION
**Pull request by marle3003**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:

- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/mokapi/ldap.d.ts
1. types/mokapi/yaml.d.ts
1. types/mokapi/faker.d.ts
1. types/mokapi/index.d.ts
1. types/mokapi/global.d.ts

<details>
<summary>as well as these 11 other files...</summary>
<p>types/mokapi/mustache.d.ts, types/mokapi/http.d.ts, types/mokapi/kafka.d.ts, types/mokapi/mokapi-tests.ts, types/mokapi/test/kafka-test.ts, types/mokapi/test/http-test.ts, types/mokapi/test/mustache-test.ts, types/mokapi/test/faker-test.ts, types/mokapi/test/global-test.ts, types/mokapi/test/ldap-test.ts, types/mokapi/test/yaml-test.ts</p>
</details>


Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 4421b5da4714d8d11e7a424ee0658e69600b1bf7
</p>

File: types/mokapi/faker.d.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+ * @param schema schema - OpenAPI Schema object contains definition of a type
+ * @example
+ * export default function() {
+ *   console.log(fake({type: 'string'}))
+ *   console.log(fake({type: 'number'}))
+ *   console.log(fake({type: 'string', format: 'date-time'}))
+ *   console.log(fake({type: 'string', pattern: '^\d{3}-\d{2}-\d{4}$'})) // 123-45-6789
+ * }
+ */
+export function fake(schema: Schema): JSONValue;
```
Comment (jakebailey) [inline types/mokapi/faker.d.ts:15]: There is no package `mokapi` on npm, but this is a module. How is this package intended to be used? What are these types for?

**Review by jakebailey — CHANGES_REQUESTED** [review]

File: types/mokapi/faker.d.ts (hunk 1 above)
Comment (marle3003) [reply types/mokapi/faker.d.ts:15]: Mokapi is a Golang app with an embedded JavaScript engine. The types are intended to provide a better developer experience when writing JavaScript code for the app.

Comment (jakebailey) [reply types/mokapi/faker.d.ts:15]: Can you provide any documentation about this?

**Review by jakebailey — APPROVED** [review]

**Comment by marle3003** [comment]
Ready to merge

**Pull request merged**